app/services/api/charts/sector.rb

```diff
@@ -141,10 +141,17 @@ def emissions_data_from_sector_benchmarks
       end
 
       def emissions_data_from_company(company)
+        assessment = company.latest_cp_assessment
+
         {
           name: company.name,
-          data: company.latest_cp_assessment&.emissions&.transform_keys(&:to_i),
-          lineWidth: 4
+          data: assessment&.emissions&.transform_keys(&:to_i),
+          zoneAxis: 'x',
```

Comment:
I know it was that before, but we should separate data layer from presentation layer somehow.